api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -90,13 +90,21 @@ type VirtualMachineClassSpec struct {
 	// policy.  The configuration specified in this field is used to customize various policies related to
 	// infrastructure resource consumption.
 	Policies VirtualMachineClassPolicies `json:"policies,omitempty"`
+
+	// Description describes the configuration of the VirtualMachineClass which is not related to virtual hardware
+	// or infrastructure policy. This field is used to address remaining specs about this VirtualMachineClass.
+	Description VirtualMachineClassDescription `json:"description,omitempty"`
```

Comment:
This can be a simple string (rather than a struct VirtualMachineClassDescription). This maps 1:1 with the vapi vmclass description, which is also a string.